frontend/workflows/projectSelector/src/project-group.tsx

```diff
@@ -224,7 +235,11 @@ const ProjectGroup: React.FC<ProjectGroupProps> = ({ title, group, displayToggle
               {!state?.loading && state?.projectData?.[key]?.linkGroups && (
                 <ProjectLinks
                   linkGroups={state?.projectData?.[key]?.linkGroups ?? []}
-                  name={[key]}
+                  onOpen={() => {
+                    onOpenQuickLinks(key);
+                  }}
+                  onClose={onCloseQuickLinks}
+                  buttonIsHidden={!(quickLinksWindowKey === key)}
```

Comment:
`button` is a bit vague here, maybe this can be something like `showOpenButton`. As a bonus, by using a positive indication in the name you can remove the negation around the evaluated boolean.